packages/venia-concept/src/components/Logo/logo.js

```diff
@@ -1,33 +1,79 @@
-import React, { Component } from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
-import classify from 'src/classify';
+import { mergeClasses } from 'src/classify';
 import logo from './logo.svg';
 
-class Logo extends Component {
-    static propTypes = {
-        classes: PropTypes.shape({
-            logo: PropTypes.string
-        }),
-        height: PropTypes.number
-    };
-
-    static defaultProps = {
-        height: 24
-    };
-
-    render() {
-        const { height, classes } = this.props;
-
-        return (
-            <img
-                className={classes.logo}
-                src={logo}
-                height={height}
-                alt="Venia"
-                title="Venia"
-            />
-        );
-    }
-}
-
-export default classify({})(Logo);
+import SmileIcon from 'react-feather/dist/icons/smile';
+import { useToastActions } from '@magento/peregrine';
+
+const Logo = props => {
+    const { height } = props;
+    const classes = mergeClasses({}, props.classes);
+
+    const { addToast } = useToastActions();
+
+    // TODO REMOVE THESE DEMO TOASTS
```

Comment:
I think I'll leave these in here so you guys can play around with them. before we merge I'll just go in and delete the code related to these demo toasts.